woocommerce-payments.php

```diff
@@ -35,6 +35,8 @@
 function wcpay_activated() {
 	// Do not take any action if activated in a REST request (via wc-admin).
```

Comment:
```suggestion
	// When WooCommerce Payments is installed and activated from the WooCommerce onboarding wizard (via wc-admin REST request), check if the site is eligible for subscriptions.
```

This comment is no longer correct when it says "do not take any action" now that there is some action taken during that flow. So I've suggested we replaced it with a tweaked version of your new comment.